src/main/java/org/mitre/fhir/authorization/ServerConformanceWithAuthorizationProvider.java

```diff
@@ -48,55 +52,61 @@ public static String getAuthorizationExtensionUri(HttpServletRequest theRequest)
     return FhirReferenceServerUtils.getServerBaseUrl(theRequest) + AUTHORIZE_EXTENSION_VALUE_URI;
   }
 
+  public static String getRevokeExtensionUri(HttpServletRequest theRequest) {
+    return FhirReferenceServerUtils.getServerBaseUrl(theRequest) + REVOKE_EXTENSION_VALUE_URI;
+  }
+
   private void fixListResource(CapabilityStatementRestComponent restComponents) {
-    restComponents
-        .getResource()
-        .stream()
-        .filter(restResource -> "List".equals(restResource.getType()))
-        .findFirst()
-        .ifPresent(listResource -> listResource.setProfile("http://hl7.org/fhir/StructureDefinition/List"));
+    restComponents.getResource().stream()
+        .filter(restResource -> "List".equals(restResource.getType())).findFirst()
+        .ifPresent(listResource -> listResource
+            .setProfile("http://hl7.org/fhir/StructureDefinition/List"));
   }
 
   @Override
   public CapabilityStatement getServerConformance(HttpServletRequest theRequest,
-                                                  RequestDetails theRequestDetails) {
+      RequestDetails theRequestDetails) {
     Extension oauthUris = new Extension();
-    oauthUris.setUrl(OAUTH_URL); //url
-
-    Extension tokenExtension = new Extension();
-    tokenExtension.setUrl(TOKEN_EXTENSION_URL);
-    UriType tokenValue = new UriType();
-    tokenValue.setValue(getTokenExtensionUri(theRequest));
-    tokenExtension.setValue(tokenValue); //valueUri
-    oauthUris.addExtension(tokenExtension);
-
-    Extension authorizeExtension = new Extension();
-    authorizeExtension.setUrl(AUTHORIZE_EXTENSION_URL);
-    UriType authorizeValue = new UriType();
-    authorizeValue.setValue(getAuthorizationExtensionUri(theRequest));
-    authorizeExtension.setValue(authorizeValue); //valueUri
-    oauthUris.addExtension(authorizeExtension);
+    oauthUris.setUrl(OAUTH_URL); // url
+
+    oauthUris.addExtension(new Extension(TOKEN_EXTENSION_URL,
+        new UriType().setValue(getTokenExtensionUri(theRequest))));
```

Comment:
```suggestion
        new UriType(getTokenExtensionUri(theRequest))));
```

Looks like there is a constructor for this too... what are the odds!